engine/table/src/test/java/io/deephaven/engine/table/impl/select/TestConditionFilterGeneration.java

```diff
@@ -31,7 +32,11 @@ public void setUp() {
                 .captureQueryScope()
                 .captureUpdateGraph()
                 .build().open();
+    }
 
+    @After
+    public void tearDown() {
+        executionContext.close();
```

Comment:
I'm not sure how we got away with not closing the exec context before, are we possibly just opening other context on top of it and never checking that none were leaked?